llvm/test/CodeGen/DirectX/typed_ptr.ll

```diff
@@ -2,9 +2,16 @@
 target triple = "dxil-unknown-unknown"
 
 ; Make sure not crash when has typed ptr.
-; CHECK:@test
+; CHECK:define i64 @test(ptr %p)
+; Make sure no bitcast generated.
+; CHECK-NOT:bitcast
```

Comment:
It'd be better to add a second test in this file (`define void @test_gep()` or so) that just does the geps and the store, rather than mixing the two, to make it clearer what the two things being tested here are.

Also, I'd probably just use update_test_checks for these rather than ad-hoc CHECK-NOTs, it's a lot clearer to see what's happening that way.

Reply:
Fixed.